helix-core/src/main/java/org/apache/helix/PropertyPathBuilder.java

```diff
@@ -53,6 +54,7 @@
 import static org.apache.helix.PropertyType.MESSAGES;
 import static org.apache.helix.PropertyType.PAUSE;
 import static org.apache.helix.PropertyType.STATEMODELDEFS;
+import static org.apache.helix.PropertyType.STATUS;
```

Comment:
Question, it seems the import method is not aligned here. Shall we use static import or just import?

Please suggest and ensure your new import aligns with the decision. I can help to quickly fix the others if you don't want to pollute this PR.

Reply:
Discussed offline. This question is about making the code style consistent in this file, without mixing usages of static/non-static.
Conclusion is: static import for the fields are fine, but preferred not too many. No static imports for classes. For this file, we'll stick to non-static imports: using `PropertyType.STATUS`. The other static imports will be addressed by #1790